index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./main.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="container">
+        <div class="logo">My Logo</div>
+        <img src="./images/hamburger.png" alt="" class="image">
+      </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, the project requires you to enclose this section under `header` tag and wrap the hamburger button with `nav`

```suggestion
    <header>
      <a href="/">My Logo</a>
      <nav>
        <img src="./images/hamburger.png" alt="" class="image">
      </nav>
    </header>
```